### ISSUE TITLE
Navigationbar not working as expected/different from the navigation buttons with trigger "validatePage"

### ISSUE THREAD

**Issue by sondre-ma** [Altinn/app-frontend-react#308]
### Description of the bug

The navigation bar lets you jump to any page as long as you have filled required fields on a page. Similarly you will not be able to navigate to _any_ page if you have not filled out every required field on the page. This creates a problem where the user will not be able to go backwards without filling out the page they are currently on. The navigation buttons works differently, and more logically, where you can always go backwards regardless of the results of the validation check, but never advance (before meeting the requirements with "validatePage").

In summary, the navigation bar should have the same logic as the buttons when navigating the form.

### Steps To Reproduce

1. Create an app (minimum two pages) with navigation buttons and a navigation bar on every page
2. Create required fields and set trigger "validatePage" on navigation components on every page
3. Start app, fill required fields on the first page, then click onto the next one
4. Without filling out the 2nd page, try navigating backwards using both the navigation bar and the buttons. These will act differently.

**Comment by bjosttveit** [comment]
This was fixed for v4, as the validation configuration for navigation bar is now separated between "going forwards" and "going backwards" similarly to how it works for navigation buttons.

https://github.com/Altinn/app-frontend-react/pull/1719
